### ISSUE TITLE
[Bug]: Problem in search for small screen

### ISSUE THREAD

**Issue by Suvadip-sana** [Soujanya2004/wanderlust-2024#237]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

As you seen in the video. The result of search is not displaying properly in mobile screen.  It's look too much annoying and not even readable format. Need to fix this

### Expected behavior

This result also supposed to be look like normal other lists.
Just like this -
![image](https://github.com/user-attachments/assets/f0cfe43c-ee1a-40b7-bbb8-6ee1e0477454)

### Add ScreenShots

Currently it display the results in this way.

https://github.com/user-attachments/assets/bbe7e27e-fa61-41a1-9e12-899f66fdffc9

@Soujanya2004  Please assign it to me. I want to fix this problem.

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

**Comment by Soujanya2004** [comment]
@Suvadip-sana this will be implemented in #236

**Comment by Suvadip-sana** [comment]
@Soujanya2004 
Yes but in your code base the search results display in seperate ejs file not in same index.ejs file. 
That's why I raised seperate ISSUE

**Comment by Soujanya2004** [comment]
@Suvadip-sana yeh ryt, i forgot
